.circleci/config.yml

```diff
@@ -43,7 +43,7 @@ jobs:
           at: ./
       - run:
           name: publish
-          command: npm publish --access public --new-version $CIRCLE_TAG
+          command: npm publish
```

Comment:
What's the reason to change to `npm publish` without specifying any `access` and `new-version` parameters?